cpp/tests/copying/scatter_tests.cpp

```diff
@@ -39,7 +39,7 @@ TEST_F(ScatterUntypedTests, ScatterMapTooLong)
   auto const source_table = cudf::table_view({source, source});
   auto const target_table = cudf::table_view({target, target});
 
-  EXPECT_THROW(cudf::scatter(source_table, scatter_map, target_table, true), cudf::logic_error);
+  EXPECT_THROW(cudf::scatter(source_table, scatter_map, target_table), cudf::logic_error);
```

Comment:
Shouldn't this no longer throw an error?
I don't think we need to test for undefined behavior.

Reply:
This is checking something slightly different. It's not verifying whether elements of the `scatter_map` are out of bounds for the `target_table`, it's checking whether there are too many elements in the `scatter_map`. That is independent of bounds checking.

Reply:
These should've been `false` originally I think then.  :-)